common/src/main/java/com/microsoft/identity/common/internal/fido/PublicKeyCredentialDescriptor.java

```diff
@@ -20,14 +20,23 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
-package com.microsoft.identity.common.internal.fido
+package com.microsoft.identity.common.internal.fido;
+
+import androidx.annotation.NonNull;
+
+import lombok.Getter;
 
 /**
  * Representation of WebAuthn's PublicKeyCredentialDescriptor.
  * https://w3c.github.io/webauthn/#dictdef-publickeycredentialdescriptor
  */
-@kotlinx.serialization.Serializable
-data class PublicKeyCredentialDescriptor(
-    val type: String,
-    val id: String
-)
+@Getter
+public class PublicKeyCredentialDescriptor {
+    public final String type;
+    public final String id;
```

Comment:
Would be a good practice to put Serialized Name annotation on these

Reply:
Added the equivalent Moshi Json annotation: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2189/commits/2e2c99898db30576eca9f758a4fbf12395573103